parsers/NL.py

```diff
@@ -211,5 +224,68 @@ def get_production_data_energieopwek(date, session=None):
 
     return df
 
+def get_wind_capacity_at(date: datetime, wind_capacity_df: pd.DataFrame) -> float:
+    in_service_mask = ((wind_capacity_df["Start date"] <= date) & (date <= wind_capacity_df["End date"])) | ((wind_capacity_df["Start date"] <= date) & (wind_capacity_df["End date"].isnull()))
+    return np.array(wind_capacity_df[in_service_mask]["#"] * wind_capacity_df[in_service_mask].kW).sum() / 1000
+
+def get_wind_capacities() -> pd.DataFrame:
```

Comment:
For this we parse an html table with for each wind turbine installation and dismantlement date